app/schema.py

```diff
@@ -60,20 +62,23 @@ class Mentee(BaseModel):
     formerly_incarcerated: bool
     underrepresented_group: bool
     low_income: bool
-    list_convictions: List[constr(max_length=255)]
+    convictions: Optional[constr(max_length=2500)]
     tech_stack: constr(max_length=255)
     job_help: bool
     pair_programming: bool
     heard_about: constr(max_length=255)
     other_info: Optional[constr(max_length=2500)]
-    validate_status: constr(max_length=255)
+    validate_status: Literal['approved', 'rejected', 'pending']
+    is_active: bool
+    in_project_underdog: bool
 
     class Config:
         extra = Extra.forbid
 
 
 class MenteeUpdate(BaseModel):
     profile_id: Optional[constr(max_length=255)]
```

Comment:
Only thing to change for MenteeUpdate is that the profile_id shouldn't be optional. Otherwise, the schema for Mentee looks great!

Reply:
Sayer made the profile_id update, and Team C has removed MentorIntake in their corresponding PR. Thanks for making these updates on the schema diagram!